doc/conf.py

```diff
@@ -43,7 +43,8 @@
     'sphinx.ext.mathjax',
```

Comment:
I guess this is a general observation.

![CleanShot 2023-10-30 at 17 05 57](https://github.com/koaning/scikit-lego/assets/1019791/85c3566b-fa67-4256-b249-a8abb5f56fd0)

Do we maybe also want to show the import path in these docs?

Reply:
It should be enough to change from:

```markdown
:::sklego.<module>.<class>
    options:
        show_root_full_path: false
        show_root_heading: true
```
to 
```markdown
:::sklego.<module>.<class>
    options:
        show_root_full_path: true
        show_root_heading: true
```
in order to render as:
![image](https://github.com/koaning/scikit-lego/assets/42817048/66df9084-5613-4b29-b36d-cae4eb1466bc)

However for some submodules it would be "partially wrong" as it would show `sklego.meta.confusion_balancer.ConfusionBalancer` while that can actually be imported as 
`from sklego.meta import ConfusionBalancer`. 

I think that keeping the full path is still better than none, yet the best solution could be to provide a (very tiny) example usage for each class.

Reply:
Most of my other open source docs tend to indeed have usage examples partially for this reason. But I'm fine with keeping that open maybe for beginner issues.

Reply:
I will open a **(good first) issue** to keep track of it